Makefile

```diff
@@ -16,6 +16,22 @@ test:
 	$(MAKE) -C pkg/policy-compiler test
 	$(MAKE) -C manager test
 
+.PHONY: run-integration-tests
+run-integration-tests: export DOCKER_HOSTNAME=kind-registry:5000
+run-integration-tests: export DOCKER_NAMESPACE=m4d-system
```

Comment:
Why are these defined here?

Reply:
as I want the default of `run-integration-tests` to work as expected using the local registry, while generally I want all other operations to work by default on the global registry (for example when building and pushing docker images or helm charts).